testing-modules/testing/src/test/java/com/baeldung/jspec/CageUnitTest.java

```diff
@@ -0,0 +1,126 @@
+package com.baeldung.jspec;
+
+import static org.javalite.test.jspec.JSpec.$;
+import static org.javalite.test.jspec.JSpec.expect;
+import static org.javalite.test.jspec.JSpec.the;
+
+import java.util.Set;
+
+import org.javalite.test.jspec.DifferenceExpectation;
+import org.junit.Test;
+
+public class CageUnitTest {
+
+    Cat tomCat = new Cat("Tom");
+    Cat felixCat = new Cat("Felix");
+    Dog boltDog = new Dog("Bolt");
+    Cage cage = new Cage();
+
+    
+    @Test
+    public void puttingAnimals_shouldIncreaseCageSize() {
```

Comment:
Let's rename the tests according to convention. For example, you could call this one "whenPutAnimalInCage_thenShouldIncreaseCageSize".